.github/workflows/cpu-long-tests.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
           # Set the OFE_SLOW_TESTS to True if running a Cron job
           OFE_SLOW_TESTS: "true"
           DUECREDIT_ENABLE: 'yes'
-          OFE_INTEGRATION_TESTS: TRUE
+          OFE_INTEGRATION_TESTS: FALSE
```

Comment:
we want CPU runner for long tests, and GPU runner for long tests + integration tests... right? @mikemhenry @IAlibay 

also why do we use `"true" 'yes' FALSE` @mikemhenry

Reply:
https://github.com/OpenFreeEnergy/openfe/blob/ecfbd65f5636f1ce311c38b679d7999cb2a940ee/openfe/tests/conftest.py#L74-L75 

looks like we check for true/false!